src/main/kotlin/org/consoletrader/rx/CandleServiceObservable.kt

```diff
@@ -0,0 +1,40 @@
+package org.consoletrader.rx
+
+import io.reactivex.Observable
+import org.consoletrader.candles.Candle
+import java.util.concurrent.atomic.AtomicReference
+
+object CandleServiceObservable {
```

Comment:
Nice work but we don't need that anymore. After ta4j is introduced that class is not required. We're putting all data to ta4j ticks collection and indicators logic is traversing the list of candles now.